src/content/admin/resource-manager.mdx

```diff
@@ -0,0 +1,127 @@
+---
+title: Resource Manager
+description: Automated resource management that adapts to changes in real utilization over time
+sidebar_label: Resource Manager
+id: resource-manager
+---
+
+import Image from "@theme/Image";
+
+Okteto's Resource Manager is an automated tool designed to optimize CPU and memory resource allocation for your Kubernetes workloads on Okteto.
+By analyzing the actual utilization of resources, the Resource Manager adjusts the resource requests for your pods, ensuring they are appropriately configured without the need for manual intervention.
+
+## Why Use the Resource Manager?
+Manually configuring CPU and memory requests in Kubernetes can be tedious and error-prone.
+Incorrect configurations can lead to poor performance, overloaded nodes, and a slow developer experience on Okteto.
+The Resource Manager automates this process, dynamically adjusting resource requests based on real-time usage data of the Okteto Cluster.
+This ensures your pods are running efficiently and your nodes are balanced well, improving overall performance and developer experience.
+
+### Key Benefits
+- **Improved Workflow Productivity**: Automates resource configuration, allowing developers to focus on coding rather than managing Kubernetes settings
+- **Optimized Resource Usage**: Ensures resources are neither underutilized nor over-utilized, leading to cost savings
+- **Enhanced Developer Adoption**: Improves the performance of development environments, leading to higher satisfaction and adoption rates amongst developers
+- **Cost Efficiency**: Reduces the overhead associated with manually configuring resources, lowering operational costs
+
+## How it Works
+
+The Resource Manager monitors the actual CPU and memory usage of your pods over time.
+It calculates the average usage and automatically adjusts the resource requests for new pods based on these averages. The system uses the following process:
+
+1. Data Collection: The Resource Manager queries the Kubernetes Metric Server to collect CPU and memory usage data for all pods managed by Okteto
```

Comment:
I would highlight than having the Metrics Server running in the cluster is a hard requirement for the RM